main.go

```diff
@@ -146,11 +198,23 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 			return
 		}
 
-		go startMonitoring(summoner, channel)
+		if watchTarget == "yamrel" {
+			s.ChannelMessageSend(m.ChannelID, "I don't watch cucks. Sorry.")
+			return
+		}
+
+		go startMonitoring(summoner, c2)
 	}
 
 }
 
+func cmdHelp(channel *discordgo.Channel, s *discordgo.Session, m *discordgo.MessageCreate) {
+	s.ChannelMessageSend(m.ChannelID, "discord-lolstatus by Grevian, bastardized by foghsho")
```

Comment:
You should probably use a single message here, with a multiline string, to avoid spamming the channel or getting rate limited
```go
helpMessage := `Some stuff
 and more stuff spread across multiple lines
using backtacks to open/close the quote`
s.ChannelMessageSend(m.ChannelID, helpMessage)
```